_build/templates/default/sass/_colors-and-vars.scss

```diff
@@ -19,8 +19,12 @@ $black: darken($colorSplash, 42.5%); /* generate the black from the $colorSplash
 $yellow: #FFEA80;
 $orange: #F49548;
 $red: #BE0000;
+$lighterRed: #FFEEEE;
 $green: #6CB24A;
+$lighterGreen: #EFFCF6;
+$greener: darken($green, 12%);
 $blue: #3697cd; /* default theme $colorSplash */
+$havelockBlue: #507dd7;
```

Comment:
```suggestion
$havelockBlue: #507DD7;
```

Improve code quality by using capitalization consistently.

Reply:
Are there editor styles available for this?

Reply:
I meant by this file: https://github.com/modxcms/revolution/blob/3.x/.editorconfig

Reply:
Open #14311